proto/internal/temporal/server/api/enums/v1/task.proto

```diff
@@ -73,3 +73,9 @@ enum TaskType {
     // Timeout task for the entire workflow execution chain.
     TASK_TYPE_WORKFLOW_EXECUTION_TIMEOUT = 29;
 }
+
+enum TaskPriority {
+    TASK_PRIORITY_UNSPECIFIED = 0;
+    TASK_PRIORITY_HIGH = 1;
+    TASK_PRIORITY_LOW = 2;
```

Comment:
nit: I'd probably keep some gap in between, so that it's possible to add new priority in the middle in the future.
e.g.: https://github.com/temporalio/temporal/blob/32212eabdd004777e094c5163df3c66a49721cce/common/tasks/priority.go#L51